.github/cache/action.yml

```diff
@@ -0,0 +1,13 @@
+name: "Cache"
```

Comment:
I'm not a fan of adding a bunch of action steps unless they'll be reused multiple times. It makes it harder to trace/follow what's going on and adds a lot of code unnecessarily (which leads to tech debt)